Add hint to application.conf changes for context to issue template (#5623)
scalableminds/webknossos: .github/ISSUE_TEMPLATE/feature_request.md

<!--- Provide a detailed description of the change or new feature you would like to have. -->

## Context
<!--- Why is this change / new feature important ? What are its use cases? -->

> * For the context the following changes to the `application.conf` are needed:
>   * [ ] Specific to long-running jobs (`jobsEnabled=true`)
>   * [ ] Specific to webKnossos.org (`isDemoInstance=true`)

Reviewer: ```suggestion
* [ ] Specific to long-running jobs (set `jobsEnabled=true` in `application.conf`)
* [ ] Specific to webKnossos.org (set `isDemoInstance=true` in `application.conf`)
```
